public/_redirects

```diff
@@ -69,3 +69,4 @@ https://www.asyncapi.io/* https://www.asyncapi.com/:splat 301!
 
 # Additional redirection
 /community/meetings /community/events 301!
+/cheatsheet https://github.com/asyncapi/website/tree/master/cheatsheet 302!
```

Comment:
Wouldn't it be nice if we add the `/cheatsheet` link as part of the Footer and README as well? It will be easily accessible for the users.

Reply:
that should be part of https://github.com/asyncapi/website/issues/2982

for now, I gave away a lot of printed cheat sheets where we have QR code to this link, and for now, it fails for people 😅